tests/helpers.py

```diff
@@ -0,0 +1,96 @@
+import pickle
+import unittest
+
+import glfw
+from mujoco_py.mjviewer import MjViewer
+import tensorflow as tf
+
+from tests.quirks import KNOWN_GYM_CLOSE_BROKEN
+from tests.quirks import KNOWN_GYM_NOT_CLOSE_VIEWER
+from tests.quirks import KNOWN_GYM_RENDER_NOT_IMPLEMENTED
+
+
+class TfTest(unittest.TestCase):
```

Comment:
I have this equivalent implementation in my PR, but also addresses memory leakage and is located in `tests/fixtures`: https://github.com/rlworkgroup/garage/pull/299/files#diff-ef6796cafe4c24fd8bb646d79094d9f6R14. Let's discuss about how to combine our implementations

Reply:
Me and Jonathon concluded that this TestCase and the one on his PR only differ in setting and removing the TF graph, so there's no big conflict for him to rebase his changes on mine.